lib/src/component2_suggestors/copyunconsumeddomprops_migrator.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:analyzer/analyzer.dart';
+import 'package:codemod/codemod.dart';
+
+/// Suggestor that updates `copyUnconsumedDomProps` and `copyUnconsumedProps` usages.
+class CopyUnconsumedDomPropsMigrator extends GeneralizingAstVisitor
+    with AstVisitingSuggestorMixin
+    implements Suggestor {
+  CopyUnconsumedDomPropsMigrator();
+
+  @override
+  visitMethodInvocation(MethodInvocation node) {
+    super.visitMethodInvocation(node);
+
+    if (node.methodName.toString() == 'addProps') {
+      var firstArg = node.argumentList.childEntities
+          .firstWhere((a) => a.toString() != '(');
+
+      if (firstArg.toString() == 'copyUnconsumedDomProps()' ||
```

Comment:
See previous PR; should use `toSource`
```suggestion
      if (firstArg.toSource() == 'copyUnconsumedDomProps()' ||
```